examples/BasicSerialLogger/BasicSerialLogger.ino

```diff
@@ -19,30 +25,23 @@ const char globalProgmemMessageWithRamString[] PROGMEM
 
 // Need to remember that backslash must be escaped in the string text
 // Banner generated with https://devops.datenkollektiv.de/banner.txt/index.html
-const char banner0[] PROGMEM = " ___   ___   ___   _ _    _                 ";
-const char banner1[] PROGMEM = "| _ \\ | __| |_  ) | | |  | |     ___   __ _ ";
-const char banner2[] PROGMEM = "|   / | _|   / /  |_  _| | |__  / _ \\ / _` |";
-const char banner3[] PROGMEM = "|_|_\\ |_|   /___|   |_|  |____| \\___/ \\__, |";
-const char banner4[] PROGMEM = "                                      |___/ ";
+const PROGMEM char banner[5][]  = { "   ___    ____   ___   ____   __             ",

```

Comment:
I needed to change this into 
````
const PROGMEM char banner[5][46]
````
otherwise on Arduino Nano I have the following compilation error:
````
..\src\../examples/BasicSerialLogger/BasicSerialLogger.ino:38:30: error: declaration of 'banner' as multidimensional array must have bounds for all dimensions except the first
 const PROGMEM char banner[5][]  = { "   ___    ____   ___   ____   __             ",
````

Reply:
Honestly I have a weak spot for ASCII art, but it'd be easier to just print a "starting \<filename> example" line (like all other RF24* examples do).

Reply:
No problem. I wanted to see how funky ASCII art can be on Arduino terminal :)